Readme.md

```diff
@@ -194,6 +194,25 @@ for filtering out certain status codes. This means that you can place your code
 handling API errors like 400's in the same places as code for handling invalid
 responses.
 
+By default, all of your requests will be put onto a background thread by `Alamofire`, and the 
+response will be called on the main scheduler.
+If you want your response called on a different thread, you can either initialize your `Provider` with a thread
+```swift
+provider = RxMoyaProvider<GitHub>(queue: DispatchQueue.global(.utility))
```

Comment:
I'm wondering if it makes sense to have a `scheduler` parameter instead of a `queue` one, to keep in line with the RxSwift semantics. This steps outside the Alamofire norms, but I think that's okay. What do you think?